app/javascript/gobierto_visualizations/webapp/visualizations/treeMapNested.vue

```diff
@@ -377,39 +383,55 @@ export default {
           })
 
         g.selectAll(".foreignobj")
-        .on('mousemove', function(d) {
+        .on('mousemove', function(d, i) {
           const { depth } = d
           if (depth === 1) return;
           let title = d.data.name === undefined ? d.data.title : d.data.name;
-          const coordinates = d3.mouse(this);
-          const x = coordinates[0];
-          const y = coordinates[1];
+          const [x, y] = d3.mouse(this);
 
           //Elements to determinate the position of tooltip
           const container = document.getElementsByClassName('container-tree-map-nested')[0];
```

Comment:
`this.$el` ?

Reply:
return `undefined`

Reply:
Return `undefined` because you're inside the function scope. You need the hack `const self = this` before the function to keep the scope.

Reply:
I know. I meant that `$el` returns undefined because `foreinObject` is created by `d3.`